res/controllers/common-controller-scripts.js

```diff
@@ -158,39 +158,6 @@ const isSimpleObject = function(obj) {
     return obj !== null && typeof obj === "object" && obj.constructor.name === "Object";
 };
 
-/**
- * Deeply merges 2 objects (Arrays and Objects only, not Map for instance).
- * @param target {object | Array} Object to merge source into
- * @param source {object | Array} Object to merge into source
- * @deprecated Use {@link Object.assign} instead
- */
-script.deepMerge = function(target, source) {
-    console.warn("script.deepMerge is deprecated; use Object.assign instead");
-
-    if (target === source || target === undefined || target === null || source === undefined || source === null) {
-        return;
-    }
-
-    if (Array.isArray(target) && Array.isArray(source)) {
-        const objTarget = target.reduce((acc, val, idx) => Object.assign(acc, {[idx]: val}), {});
-        const objSource = source.reduce((acc, val, idx) => Object.assign(acc, {[idx]: val}), {});
-        deepMerge(objTarget, objSource);
-        target.length = 0;
-        target.push(...Object.values(objTarget));
-    } else if (isSimpleObject(target) && isSimpleObject(source)) {
```

Comment:
You can remove the `isSimpleObject` function above.

Reply:
Thanks! Done.